docs/Sprint-Review/sprint-25-summary.md

```diff
@@ -0,0 +1,70 @@
+# Sprint 25 Summary
+**07/06/2021- 07/20/2021**
+
+### Summary Updates
+- The Engineering team continued to work on Secret Key Epic tickets and various devops related tickets to finishing out ATO functionality as well as accessibility work for Django admin and other areas.
+- The UX Research team focused on Parsing Error blockers and understanding Regional Staff user journeys, and began investigations into the user access request journey
+- NextGen XMS and ACF AMS teams have given us technical information in order to assess and estimate our technical implimentation and recommendation to TDP Product Owner and tech lead. Our next task is to write integration tickets and estimate this work and provide a recommendation to OFA.
+
+
+## Sprint 25 Goals
+- Finish closing ATO functionality tickets
+- Focus on Secret Key Leakage Mitigation Epic
+- UX Ideation for Parsing Blocker Communications
+
+**Next Sprint (26) Goals:** 
```

Comment:
```suggestion
**[Next Sprint (26) Goals](https://github.com/raft-tech/TANF-app/milestone/29)** 
```

Instead of revising in both places, lets just link to Milestone goals